oidc-controller/api/core/config.py

```diff
@@ -167,6 +167,15 @@ class GlobalConfig(BaseSettings):
         "CONTROLLER_PRESENTATION_EXPIRE_TIME", 10
     )
 
+    # How long auth_sessions with matching the states in
+    # CONTROLLER_SESSION_TIMEOUT_CONFIG_FILE are stored for in seconds
+    CONTROLLER_PRESENTATION_CLEANUP_TIME: int = os.environ.get(
+        "CONTROLLER_PRESENTATION_CLEANUP_TIME", 86400
+    )
+
+    CONTROLLER_SESSION_TIMEOUT_CONFIG_FILE: Optional[str] = os.environ.get(
+        "CONTROLLER_SESSION_TIMEOUT_CONFIG_FILE"
+    )
```

Comment:
This assumes there MUST be a file with the configuration. Can we provide either a default file or a set of default values to use so the service can be run as-is?

Reply:
A config file is not necessary the user will just be warned that no ttl will be applied. I can still add defaults though. I figured a safe default was not to delete anything by default but instead I can just make it use any non verified auth session.

Reply:
I should have read past the first line of the log statement 🙂 
An update to the docs and we should be good then!

Reply:
Just added documentation. Thanks